_posts/Rowan-Loft/2022-11-10-initial-idea-and-work-plan.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+author: Rowan-Loft
+title: Rowan-Loft's Initial Idea & Work Plan
+---
+
+For my final project, I will be looking to create a user-navigable program for a sample size of public commercial and (tentatively) non-commercial aiports in the United States. I say a sample size because there are over 5,000 public airports, and this program will specifically not include all other airports in the US, as the number jumps to just under 25,000.
+
+There is already a present and up to date .csv containing data that provides information on open airports, closed airports, and helipads in the US that is produced by the United Nations Office for the Coordination of Human Affairs (OCHA), The list will have to be trunkated to some extent as I'm not interested in closed airports and helipads, which contribute to the files' just-under 30,000 rows of data. Further cutting will have to be done to account for some information that would not be pertinent to someone navigating the program such as elevation, longitude, and latitude of an airport. 
```

Comment:
typo: '.' after (OCHA)-Hannah

Reply:
Also: truncated

Reply:
Fixed and fixed. Thanks for keeping an eye out.